src/Core/src/Platform/MaciOS/DateExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Foundation;
+
+namespace Microsoft.Maui
+{
+	public static class DateExtensions
+	{
+		public static DateTime ToDateTime(this NSDate date)
+		{
+			return new DateTime(2001, 1, 1, 0, 0, 0).AddSeconds(date.SecondsSinceReferenceDate);
```

Comment:
Can we make 2001-1-1 static here, instead of creating a new DateTime(2001, 1, 1, 0, 0, 0) every time we convert?

Reply:
Good point, changed.